### ISSUE TITLE
[Feature]: Make Region Emum a backed enumeration

### ISSUE THREAD

**Issue by tacman** [ToshY/BunnyNet-PHP#130]
### Description

After I fetch the zones from the API, I have a string for the region, e.g. 'NY'  Now when I call the edit API, I need to pass in the region enum, but there's no way to get enum from the string.

I think the solution is simply a backed enum: https://www.php.net/manual/en/language.enumerations.backed.php

### Example

```php
$edgeStorageApi = new EdgeStorageAPI(
    apiKey: '6bf3d93a-5078-4d65-a437-501c44576fe6',
    region: Region::FS,
    client: $bunnyClient,
);

// OR

$edgeStorageApi = new EdgeStorageAPI(
    apiKey: '6bf3d93a-5078-4d65-a437-501c44576fe6',
    region: Region::from('FS'),
    client: $bunnyClient,
);
```

**Comment by ToshY** [comment]
Hey @tacman 👋 

Thanks for the suggestion. I'm a fan of backed enums so this sounds like a good idea.

I will create a PR later today (and mention you again when it's ready).

**Comment by ToshY** [comment]
@tacman This has been released in [4.4.0](https://github.com/ToshY/BunnyNet-PHP/releases/tag/4.4.0).
